rust/rbac-registration/src/cardano/cip509/cip509.rs

```diff
@@ -454,7 +454,7 @@ fn payment_history(
     report: &ProblemReport,
 ) -> HashMap<ShelleyAddress, Vec<Payment>> {
     let hash = MultiEraTx::Conway(Box::new(Cow::Borrowed(txn))).hash();
-    let context = format!("Populating payment history for Cip509, transaction hash = {hash:?}");
+    let context = format!("Populating payment history for Cip509, transaction hash = {hash}");
```

Comment:
Same as bellow, don't mention "hash" end users won't know what that means.
Talk about the "Payment Address" and use the format I describe below (cip 19 with a hash (0x...) following it.